lib/sycamore/sycamore/connectors/duckdb/duckdb_writer.py

```diff
@@ -125,23 +120,22 @@ def create_target_idempotent(self, target_params: BaseDBWriter.TargetParams):
                         dtype += f"[{dict_params.get('dimensions')}]"
                     columns.append(f"{key} {dtype}")
                 columns_str = ", ".join(columns)
-                client.sql(f"CREATE TABLE {dict_params.get('table_name')} ({columns_str})")
+                self._client.sql(f"CREATE TABLE {dict_params.get('table_name')} ({columns_str})")
             else:
                 logging.warning(
                     f"""Error creating table {dict_params.get('table_name')}
                     in database {dict_params.get('db_url')}: no schema provided"""
                 )
         except Exception:
-            return
+            logging.info(f"Table {dict_params.get('table_name')} already exists")
```

Comment:
are there any other errors that could come up here? Maybe change the log to something like "Table <table> could not be created. most likely it already exists". Might also make it a debug level or something since you'll get hundreds of these on big jobs